### PR TITLE
Updated canonicals for prob1/prob3

### PR DESCRIPTION
**Pull request by vkbo**
Updated canonicals for prob1/prob3

Don't merge yet. Still waiting for the fort.90s

### CODE REVIEW

**Comment by vkbo** [comment]
Unless you have the fort.90s @kyrsjo ?

**Comment by kyrsjo** [comment]
Hi, yes, I just mad a branch with that. But I can rather put them on your branch instead. 2 sec...

File: .gitattributes [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-lib/zlib/* linguist-vendored
+# Setup for github-linguist
+lib/zlib/*  linguist-vendored
 
 *.canonical linguist-generated
-test/*/fort.2 linguist-generated
-*.8 linguist-generated
-test/*/fort.13 linguist-generated
+*.2         linguist-generated
+*.3         linguist-language=text
```
Comment (kyrsjo) [inline .gitattributes]: good idea!

**Comment by kyrsjo** [comment]
We also need to update fort.10. I'll push those too.

**Comment by kyrsjo** [comment]
Done!

**Pull request merged**